obs/tag.js

```diff
@@ -1,10 +1,13 @@
 const { Struct } = require('mutant')
 
-module.exports = function(server, api) {
-  return function(tagId) {
+// Not quite sure this is useful with the patchcore stipped out.
+// Could be we roll our own with ssb-about
```

Comment:
Are you suggested implementing the `about.obs.name` functionality here? Or perhaps moving functions like `socialValue` and `valueFrom` to ssb-about and access through the server?

Reply:
either the interface will implement it (e.g. if you're using patchcore you can drop in about.obs.name) or yeah I think it could be a job of the server - this is exactly what Dominic has done with [ssb-names](https://github.com/ssbc/ssb-names) (note the module does more than the out of date readme suggests!)